public/js/chat.js

```diff
@@ -37,14 +42,16 @@
   socket.on('newMessage', (data) => {
     let msgs = document.querySelector("#msgs");
     let template;
-    if(socket.id == data.senderId){
-      template = `<div class="one column row msg"><div class="right floated purple seven wide column">${data.message}</div></div><br>`;
+    let time = moment().format('LT');
+
```

Comment:
just make changes what you were told to. pr should be related to the changes you make. these brackets and space changes are not meant to be done in this one.